src/main/java/com/sda/util/HibernateUtil.java

```diff
@@ -1,4 +1,24 @@
 package com.sda.util;
 
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
 public class HibernateUtil {
+    private static final SessionFactory sessionFactory;
+
+    static {
+        try {
+            // Create the SessionFactory from standard (hibernate.cfg.xml)
+            // config file.
+            sessionFactory = new Configuration().configure("/com/sda/config/hibernate.cfg.xml").buildSessionFactory();
```

Comment:
If we use **resources** directory then we do not need to specify an address string argument for **configure()** method